lib/Models/Terria.ts

```diff
@@ -564,7 +563,8 @@ export default class Terria {
           .filter((v7initUrl: any) => isJsonString(v7initUrl))
           .map(async (v7initUrl: string) => {
             const catalog = await loadJson5(v7initUrl);
-            const convert = convertCatalog(catalog);
+            const { convertCatalog } = await import("catalog-converter");
+            const convert = convertCatalog(catalog, { generateIds: false });
```

Comment:
Just a suggestion. We could maybe use `Promise.all` here to do the load & import requests in parallel? Maybe even extract it out as an async function called `convertV7Catalog` if the resulting code is a bit dense.

Reply:
Done